src/main/webapp/challenges/vc9b78627df2c032ceaf7375df1d847e47ed7abac2a4ce4cb6086646e0f313a4.jsp

```diff
@@ -82,8 +82,8 @@
 					<tr>
 						</td>
 						<select id='userId' style='width: 300px;' multiple>
-							<option value="c81e728d9d4c2f636f067f89cc14862c">Paul
-								Bourke</option>
+							<option value="c81e728d9d4c2f636f067f89cc14862c">Joe
+								Sullivan</option>

```

Comment:
Not sure if this change is necessary?

Reply:
The userid `c81e728d9d4c2f636f067f89cc14862c` belongs to Joe Sullivan, not Paul Burke. It's just a small logical error, nothing really important. Nevertheless, I propose to correct this.

Reply:
Makes sense now, thanks.